app/assets/v2/js/notifications.js

```diff
@@ -111,24 +111,26 @@ Vue.mixin({
 
 });
 
-var app = new Vue({
-  delimiters: [ '[[', ']]' ],
-  el: '#gc-notifications',
-  data: {
-    page,
-    notifications,
-    unreadNotifications,
-    hasNext,
-    numPages,
-    numNotifications
-  },
-  mounted() {
-    this.fetchNotifications();
-  },
-  created() {
-    this.sendState();
-  }
-});
+if (document.getElementById('gc-notifications')) {
```

Comment:
Use jquery selector?

Reply:
Im trying to avoid jquery as a dependence on that js, maybe for a near future (next version of bootstrap jquery will be removed) The idea is also if we want to use that to make an extension, work with service workers or whatever We are not depend on jquery in notifications.